azure-mgmt-storage/src/main/java/com/azure/management/storage/models/BlobServicesInner.java

```diff
@@ -181,23 +184,23 @@ public BlobServicePropertiesInner setServiceProperties(String resourceGroupName,
     /**
      * Gets the properties of a storage account’s Blob service, including properties for Storage Analytics and CORS (Cross-Origin Resource Sharing) rules.
      * 
-     * @param resourceGroupName 
-     * @param accountName 
+     * @param resourceGroupName The name of the resource group within the user's subscription. The name is case insensitive.
+     * @param accountName The name of the storage account within the specified resource group. Storage account names must be between 3 and 24 characters in length and use numbers and lower-case letters only.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
      * @throws CloudException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<SimpleResponse<BlobServicePropertiesInner>> getServicePropertiesWithResponseAsync(String resourceGroupName, String accountName) {
         final String blobServicesName = "default";
-        return service.getServiceProperties(this.client.getHost(), resourceGroupName, accountName, this.client.getSubscriptionId(), blobServicesName, this.client.getApiVersion());
+        return service.getServiceProperties(this.client.getHost(), resourceGroupName, accountName, this.client.getApiVersion(), this.client.getSubscriptionId(), blobServicesName);

```

Comment:
this raises a little concern, curious why order of params changed

Reply:
change in modelerfour. currently there is no strict order on parameters, only non-required parameter after required parameter.

Reply:
Also since apiVersion and subscriptionId is filled internally, the method signature remain unchanged as the public client method. (method is service is not public hence user cannot use them)